spiped/dispatch.c

```diff
@@ -27,6 +27,7 @@ struct accept_state {
 	size_t nconn_max;
 	double timeo;
 	void * accept_cookie;
+	void * event_cookie;
```

Comment:
`event_cookie` is a bit vague, since every event (network, timer, immediate) we can schedule provides a cookie.  How about `dnstimer_cookie`?